cmake/thirdParties.cmake

```diff
@@ -4,6 +4,9 @@
 
 set(BLA_DEFINITIONS)
 
+set(BLA_VENDOR "OpenBLAS" CACHE STRING "BLAS/LAPACK implementation")
+# cannot set this as travis sometimes use Intel10_64lp ...
+#set_property(CACHE BLA_VENDOR PROPERTY STRINGS OpenBLAS Intel ATLAS Apple)
```

Comment:
to remove then?

Reply:
yes... unless you want to turn your travis variable Intel10_64lp into Intel (other cases ?...)

Reply:
thinking about it I'm also -1 for this one. Setting `BLA_VENDOR` to something that you don't have causes a more nonesense error down the line.

it forces the user to set it.

Maybe what we should do is to keep it this way (unset) and write a nicer error message asking the user to take the time to think what does he want to compile and pass it properly. 

IMHO not setting things so that they carsh on the right time or force to pass parameters, is a feature not a bug.